lib/p2p/NodeList.ts

```diff
@@ -233,13 +244,14 @@ class NodeList extends EventEmitter {
     const { nodePubKey } = node;
     const alias = pubKeyToAlias(nodePubKey);
     if (this.aliasToPubKeyMap.has(alias)) {
-      throw errors.ALIAS_CONFLICT(alias);
+      this.aliasToPubKeyMap.set(alias, 'CONFLICT');
```

Comment:
I'm assuming that you are doing this for a good reason but if possible there should just be an error thrown when the conflict is discovered

Reply:
Originally there was an error thrown here but that's one thing this PR is trying to change. We don't want to prevent ourselves from connecting to a node just because it has an alias conflict with another node. We will throw errors if we try to use a conflicted alias for RPC calls, however.